commands/members/add.js

```diff
@@ -3,36 +3,76 @@
 let cli = require('heroku-cli-util')
 let co = require('co')
 let extend = require('util')._extend
+let Utils = require('../../lib/utils')
 
 function * run (context, heroku) {
-  // Users receive `You'll be billed monthly for teams over 5 members.`
-  // message when going over the FREE_TEAM_LIMIT
-  const FREE_TEAM_LIMIT = 5
-  let org = context.org
+  let orgInfo = yield Utils.orgInfo(context, heroku)
+  let orgName = context.org
+
+  const warnMembershipLimit = function * (totalMembers) {
+    // Users receive `You'll be billed monthly for teams over 5 members.`
+    if (totalMembers === 6) {
+      cli.warn("You'll be billed monthly for teams over 5 members.")
+    }
+  }
+
+  let addMemberToOrg = function * (email, role, orgName) {
+    let request = heroku.request({
+      method: 'PUT',
+      path: `/organizations/${orgName}/members`,
+      body: {email, role}
+    })
+    yield cli.action(`Adding ${cli.color.cyan(email)} to ${cli.color.magenta(orgName)} as ${cli.color.green(role)}`, request)
+  }
+
+  let inviteMemberToTeam = function * (email, role, orgName) {
+    let request = heroku.request({
+      headers: {
+        Accept: 'application/vnd.heroku+json; version=3.team-invitations'
+      },
+      method: 'PUT',
+      path: `/organizations/${orgName}/invitations`,
+      body: {email, role}
+    }).then(request => {
+      cli.action.done('email sent')
+    })
+
+    yield cli.action(`Inviting ${cli.color.cyan(email)} to ${cli.color.magenta(orgName)} as ${cli.color.green(role)}`, request)
+  }
+
   let email = context.args.email
   let role = context.flags.role
-  let features = yield heroku.get('/account/features')
-  let orgCreationFeature = features.find(function (feature) {
-    return feature.name === 'standard-org-creation'
-  })
-
-  let request = heroku.request({
-    method: 'PUT',
-    path: `/organizations/${org}/members`,
-    body: {email, role}
-  })
-  yield cli.action(`Adding ${cli.color.cyan(email)} to ${cli.color.magenta(org)} as ${cli.color.green(role)}`, request)
-
-  let members = yield heroku.get(`/organizations/${org}/members`)
-  if ((members.length === (FREE_TEAM_LIMIT + 1)) && orgCreationFeature) {
-    cli.log("You'll be billed monthly for teams over 5 members.")
+
+  let groupFeatures = yield heroku.get(`/organizations/${orgName}/features`)
+
+  if (groupFeatures.filter(of => of.name === 'team-invite-acceptance').length) {
+    yield inviteMemberToTeam(email, role, orgName)
+  } else {
+    yield addMemberToOrg(email, role, orgName)
+  }
+
+  if (orgInfo.type === 'team') {
```

Comment:
also, I don't want to show that warning to enterprise orgs (which don't have limits) regardless of the `team-invite-acceptance` flag.